### PR TITLE
Refactoring screen-arrange Part 1: Sidebar

### PR DESCRIPTION
**Pull request by owehmer**
For easier refactoring in the future I want the every visible component to be their own Angular component.

I removed some unused CSS classes aswell.

I had to copy some styles for now, but they will most likely be deleted anyway when the refactoring is done.

### CODE REVIEW

**Comment by owehmer** [comment]
@negue can you tell me what the items array is being used for? 😆 i copied it for now, but I don't think it makes a lot of sense 😆 

```ts
items: string[] = ['Extra cheese', 'Mushroom', 'Onion', 'Pepperoni', 'Sausage', 'Tomato'];
```

**Comment by negue** [comment]
hahahaha  you found my magic recipe  :D 

I think it was from the example drag&drop library haha - nice easter egg

**Comment by owehmer** [comment]
Do you want me to delete it or let it in? :D

File: src/app/shared/dialogs/screen-arrange/screen-arrange-sidebar/screen-arrange-sidebar.component.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  @Input()
+  screen: Screen;
+
+  @Input()
+  currentSelectedClip: CombinedClip | null = null;
+
+  @Output()
+  changeCurrSelectedClip = new EventEmitter<CombinedClip | null>();
+
+  items: string[] = ['Extra cheese', 'Mushroom', 'Onion', 'Pepperoni', 'Sausage', 'Tomato'];
```
Comment (negue) [inline src/app/shared/dialogs/screen-arrange/screen-arrange-sidebar/screen-arrange-sidebar.component.ts]: can be removed :P

**Review by negue — APPROVED** [review]
Ok soo I only found 2 issues:

1. the layout / placing of the sidebar is now different but I think this can be pushed to a different pr
2. the selected border color is gone on the media card now

otherwise good to merge 👍

File: src/app/shared/dialogs/screen-arrange/screen-arrange-sidebar/screen-arrange-sidebar.component.ts (hunk 1 above)
Comment (negue) [reply src/app/shared/dialogs/screen-arrange/screen-arrange-sidebar/screen-arrange-sidebar.component.ts]: `easter is now officially over folks.` haha 👍

**Comment by negue** [comment]
Perfect! Thank you very much  - awesome that helps to improve it .. and can't wait for the next parts :D

**Comment by owehmer** [comment]
>the layout / placing of the sidebar is now different but I think this can be pushed to a different pr

i found a missing h3 style. i think it looks identical to dev now

**Pull request merged**